app/controllers/api/v2/site_statuses_controller.rb

```diff
@@ -7,6 +7,10 @@ def update
         site_status = authorize SiteStatus.find(params[:id])
         site_status.update site_status_params
 
+        if site_status.vacancies_filled?
```

Comment:
not what I had in mind I was thinking of https://api.rubyonrails.org/classes/ActiveModel/Dirty.html
```suggestion
        if site_status.vac_status_changed?(to: "no_vacancies")
```
if inline or probably better belongs in the site_status model

you probably do want to keep the `running` guard which you added

Reply:
https://github.com/DFE-Digital/teacher-training-api/pull/1439/files#diff-1d30bc581333157c3ffe153a1c241349R88